.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,7 +18,7 @@ Fixes #xyz
 
 <!-- or this PR is one task of an issue -->
 
-Master Issue: #xyz
+Parent Issue: #xyz
```

Comment:
```suggestion
Main Issue: #xyz
```

If it's because GitHub's attitude, why not `main`? Parent is somehow unintuitive.